examples/network-hhmodel/incidence_report.rs

```diff
@@ -0,0 +1,54 @@
+use ixa::context::Context;
+use ixa::error::IxaError;
+use ixa::global_properties::ContextGlobalPropertiesExt;
+use ixa::people::PersonPropertyChangeEvent;
+use ixa::report::ContextReportExt;
+use ixa::ContextPeopleExt;
+use ixa::{create_report_trait, report::Report};
+use std::path::PathBuf;
+
+use serde::{Deserialize, Serialize};
+
+use crate::parameters::Parameters;
+use crate::seir::{DiseaseStatus, DiseaseStatusValue, InfectedBy};
+
+#[derive(Serialize, Deserialize, Clone)]
+struct IncidenceReportItem {
+    time: f64,
+    person_id: String,
+    infection_status: DiseaseStatusValue,
+    infected_by: String,
+}
+
+create_report_trait!(IncidenceReportItem);
+
+fn handle_infection_status_change(
+    context: &mut Context,
+    event: PersonPropertyChangeEvent<DiseaseStatus>,
+) {
+    context.send_report(IncidenceReportItem {
+        time: context.get_current_time(),
+        person_id: event.person_id.to_string(),
+        infection_status: event.current,
+        infected_by: context
+            .get_person_property(event.person_id, InfectedBy)
+            .unwrap()
```

Comment:
Yes, this would cause a panic if it is `None` so as I commented in `main.rs` you can use `unwrap_or` so you can provide a default for when the person is one of the index cases in the population.  For example,


```suggestion
            .unwrap_or("NA")
```

could be a possible way to handle.

Reply:
We do provide a default for the index person. 

And .unwrap_or() is not compiling for me here. I can look into other options, but if the infected_by person property was empty and being added to the report, we may want it to panic? The point is that each infected person gets a infected_by entry. Will defer to you all!

Reply:
Ok changed 2 things to account for the issue of possibly empty values here: 
1. added a condition so that the only time something is getting added to this report is when they change from S->E (not other changes to DiseaseStatus). When they change from S->E, they get InfectedBy value
2. added an error handling for if for some reason, the infectedBy value is somehow still empty. Resolving this comment for now :)